test/unit/auth/auth-api-request.spec.ts

```diff
@@ -2404,6 +2408,1017 @@ describe('FirebaseAuthRequestHandler', () => {
     });
   });
 
+  describe('getOAuthIdpConfig()', () => {
+    const providerId = 'oidc.provider';
+    const path = `/v2beta1/projects/project_id/oauthIdpConfigs/${providerId}`;
+    const method = 'GET';
```

Comment:
Nit: Calling this `getMethod` or `httpGet` would make the rest of the code more readable. Or even using the string literal as `callParams(path, 'GET', {})` is not a bad idea.